parcels/codegenerator.py

```diff
@@ -761,10 +761,11 @@ def visit_SummedVectorFieldEvalNode(self, node):
         for U, V, W, var, var2, var3 in zip(node.field.obj.U, node.field.obj.V, Wlist, node.var, node.var2, node.var3):
             vfld = VectorField(node.field.obj.name, U, V, W)
             ccode_eval = vfld.ccode_eval(var, var2, var3, U, V, W, *node.args.ccode)
-            ccode_conv1 = U.ccode_convert(*node.args.ccode)
-            ccode_conv2 = V.ccode_convert(*node.args.ccode)
-            statements = [c.Statement("%s *= %s" % (var, ccode_conv1)),
-                          c.Statement("%s *= %s" % (var2, ccode_conv2))]
+            if U.interp_method != 'cgrid_velocity':
+                ccode_conv1 = U.ccode_convert(*node.args.ccode)
+                ccode_conv2 = V.ccode_convert(*node.args.ccode)
+                statements = [c.Statement("%s *= %s" % (var, ccode_conv1)),
+                              c.Statement("%s *= %s" % (var2, ccode_conv2))]
```

Comment:
an `else` part is missing (see other vector meta fields). Otherwise now `statement` is not defined for A-grids